site/src/patterns/message.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Chat
```

Comment:
Is a chat and a message the same thing? If so, should this be the message pattern?

Reply:
updated